r4/metadata/modules/utils/utils_common.bal

```diff
@@ -14,14 +14,32 @@
 // specific language governing permissions and limitations
 // under the License.
 
-public enum IssueType {
-    INVALID = "invalid",
-    REQUIRED = "required",
-    NOT_FOUND = "not-found",
-    CODE_INVALID = "code-invalid",
-    INFORMATIONAL = "informational",
-    EXCEPTION = "exception",
-    TIMEOUT = "timeout",
-    PROCESSING = "processing",
-    VALUE = "value"
+import ballerina/log;
+
+# Debug logger.
+# 
+# + msg - debug message 
+public isolated function LogDebug(string msg) {
+    log:printDebug(msg);
+}
+
+# Error logger.
+#
+# + err - error message
+public isolated function LogError(error err) {
+    log:printError(string `message: ${err.detail().toString()}, stacktrace: ${err.stackTrace().toString()}`);
```

Comment:
What is the exact requirement here? If we need to log the error message, it can be accessed using `err.message()`. Then to print stacktrace, we can print it using a separate key. 

```suggestion
    log:printError(err.message()}, stacktrace = err.stackTrace());
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/9e0cd675493486c9bb967139981edde086bbb830